tests/unittest/conftest.py

```
@@ -216,6 +216,12 @@ def pytest_sessionstart(session):
         os.environ["TLLM_DISABLE_MPI"] = "1"
         os.environ["TLLM_RAY_FORCE_LOCAL_CLUSTER"] = "1"
 
+    # Setup Triton cache directory to avoid "No such file or directory" errors
+    from pathlib import Path
+    triton_cache_dir = os.environ.get('TRITON_CACHE_DIR', '/tmp/triton_cache')
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/triton_cache"

(S108)